js/ct-checkusers.js

```diff
@@ -135,6 +144,7 @@ window.apbct = window.apbct || {};
     };
 
     usersChecker.runUserCheckerSuccess = (response) => {
+        usersChecker.reactUserCheckingFrontendData(response?.checkingCount, response?.foundedSpam);
```

Comment:
Is there a way not using the `optional chaining '?.'` functioning ?

Reply:
DONE